README.md

```diff
@@ -169,6 +169,129 @@ docker inspect docker-beegfs-demo_beegfs
 
 Now you can connect to the management node from another machine using its ip address.
 
+## Setup on non-virtual (physical) machine (Ubuntu 22.04) && Potential troubleshooting:
+At first, you should follow previously mentioned instructions. Whether there are problems still, you can try the following fixes.
+
+There are few tricky moments that aren’t mentioned here and very poorly mentioned on the internet. The thing is, the physical machine with docker **MUST** be connected via ethernet. Whether it is connected via wifi, it is likely that the addresses of management/metadata/storage nodes couldn’t be pinged.
+
+Another important note is that the client-machine (M1 macOS Sonoma in my case) **MUST NOT** be connected via ethernet, but with wifi instead. For some reason, when both computers use ethernet, the docker is inaccessible.
+
+Lastly, the output of `ip -o -f inet ` which is used to find the subnet, will return you, for example: `192.168.1.26/24 `. If nothing works, you might try the following in docker-compose.yml (subnet) `192.168.1.0/24 `.
+
+## Manual images rebuild
+Whether you need to edit the beegfs-[SERVICE].conf file and apply the changes, you must do some additional steps rather than only editing the conf files:
+1.	Stop the container:
+```sh
+docker-compose down
+```
+2.	Check the existing images ID:
+```sh
+docker images
+```
+3.	Delete all related images:
+```sh
+docker images rmi -f [IMAGE_ID]
+```
+Use the following docker-compose.yml file (don’t forget put on your parent, subnet, ip_range and gateway)
+
+```yml
+version: "2"
+ 
+services:
+  management:
+    image: beegfs-demo-management
+    build: 
```

Comment:
It seems the original `docker-compose.yml` will work for build, it just requires adding `build` and `context` for each service:
```yml
     build:
       context: management
```
So let's change the original `docker-compose.yml` and remove this one from the README.